bot/tools/restart_tasks.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+import argparse
+import os
+
+from taskcluster.utils import slugId
+
+from code_coverage_bot.secrets import secrets
+from code_coverage_bot.taskcluster import taskcluster_config
+
+CODECOV_URL = 'https://codecov.io/api/gh/marco-c/gecko-dev/commit'
+MC_REPO = 'https://hg.mozilla.org/mozilla-central'
+HOOK_GROUP = 'project-relman'
+HOOK_ID = 'code-coverage-{app_channel}'
+
+taskcluster_config.auth(
+    os.environ.get('TASKCLUSTER_CLIENT_ID'),
+    os.environ.get('TASKCLUSTER_ACCESS_TOKEN'),
+)
+secrets.load(
+    os.environ['TASKCLUSTER_SECRET'],
+)
+queue = taskcluster_config.get_service('queue')
+
+
+def list_commits(tasks):
+    '''
+    Read the revision from an existing code coverage task
+    '''
+    for task_id in tasks:
+        try:
+            task = queue.task(task_id)
+            yield task['payload']['env']['REVISION']
+        except Exception as e:
+            print('Failed to load task {}: {}'.format(task_id, e))
+
+
+def trigger_task(task_group_id, commit):
+    '''
+    Trigger a code coverage task to build covdir at a specified revision
+    '''
+    assert isinstance(commit, str)
+    name = 'covdir {} - {}'.format(secrets[secrets.APP_CHANNEL], commit)
+    hooks = taskcluster_config.get_service('hooks')
+    payload = {
+        'REPOSITORY': MC_REPO,
```

Comment:
Could we read the repo too from the failing task? This way we support retriggering both MC and try tasks

Reply:
Done